components/formats-gpl/src/loci/formats/in/ZeissCZIReader.java

```diff
@@ -1363,6 +1328,18 @@ else if (getZCTCoords(plane)[2] < timestamps.size()) {
     segments = null;
   }
 
+  // -- ZeissCZI-specific methods --
+
+  public boolean allowAutostitching() {
+    return ((DefaultMetadataOptions) getMetadataOptions()).getBoolean(
```

Comment:
@melissalinkert: I think we want to sort this in a relevant call before taking any action. We can certainly update all usages of MetadataOptions in a go once we make the final decision on the API/inheritance pattern for 5.3.0.